frontend/src/app/modules/posts/sevices/post.service.ts

```diff
@@ -99,10 +76,8 @@ export class PostService {
         this.isLoadingGet.set(false);
       }),
       tap(
-        (data) => {
-          this.postsSubject.next(data.posts);
-        },
-        (error) => {this.alertService.showAlert('Something went wrong...', 'error'); console.log(error)},
+        (data) => { console.log(data)},
+        (err) => {this.alertService.showAlert('Something went wrong due to getting posts', 'error');}
```

Comment:
troche dziwnie to napisane, będę robić funkcję do handlowania błędów ale i tak mógłbyś to poprawić

Reply:
co masz na mysli konkretnie ?

Reply:
dałbym "while getting posts" zamiast "due to"